api/jest-ts-preprocessor.js

```diff
@@ -0,0 +1,11 @@
+const tsc = require('typescript');
+const tsConfig = require('./tsconfig.json');
+
+module.exports = {
+  process(src, path) {
+    if (path.endsWith('.ts') || path.endsWith('.tsx')) {
```

Comment:
Is this check required? I'd think the regex in `transform` would cover it.

Reply:
Uhhhh good question. To be honest I just copy-pasted this from an [official jest example](https://github.com/facebook/jest/blob/master/examples/typescript/preprocessor.js), so I'm not sure. It does make sense that the regex would cover it...